### ISSUE TITLE
SQL Engine accepts double where clause in a single statement

### ISSUE THREAD

**Issue by guglielmomoretti** [orientechnologies/orientdb#4495]
Good Morning.

By mistake I wrote the following query in OrientDB Studio:

```sql
select from TransitsSummary where code = 'NC' where year = 2015
```

As you can see, there are **two** where clauses on the query. 
The query result shows the following:

![image](https://cloud.githubusercontent.com/assets/2829667/8449419/93ea2a02-1fd1-11e5-8e89-c78676a9485d.png)

As you can see from the picture, the query **"works"**, in the sense that gives out results. 

Is it expected to have two where clauses in your SQL dialect?

*(however the engine seems to take only in account the last where in the query. Actually It is filtering by year but not by code)*

Regards
Guglielmo Moretti

**Comment by luigidellaquila** [comment]
Hi Guglielmo,

I guess you are trying it on 2.0.x release. This problem should be fixed in 2.1 snapshot release. Could you please confirm?

Thanks

**Comment by guglielmomoretti** [comment]
Hi Luigi,

I'm sorry, I forgot to mention the version.
Yes, I'm running OrientDB ver. 2.0.11.

I'll let you know as soon I try the snapshot release.

Regards
Guglielmo

**Comment by lvca** [comment]
This has been already fixed since 2.1-rc1.
